### ISSUE TITLE
Add support for classifiers for jar

### ISSUE THREAD

**Issue by hc-codersatlas** [luben/zstd-jni#63]
Many builds with natives have support for maven classifiers (used to provide separate jars depending on OS). This reduces download size etc. Would be good to add this as more and more platforms are added.

**Comment by luben** [comment]
Seems reasonable, do you have an example I can look at?

**Comment by hc-codersatlas** [comment]
Sure, https://github.com/netty/netty-tcnative/blob/master/pom.xml is 1 example from Netty.

You can check the classifier related parts.

https://geekzguru.wordpress.com/2008/02/16/maven-tag/ is an article explaining the maven tag. 

Sorry, not familiar with sbt to help on that though.

**Comment by luben** [comment]
Yes, I will look how to provide alternative jars with classifiers.

**Comment by luben** [comment]
I made the build.sbt changes and it looks they are producing correct jars.  Verified by "publishLocal". With the next release I will start publishing them to maven central.

**Comment by luben** [comment]
zstd-jni-1.3.4-11 just landed on Maven Central with the classified jars.
